README.md

```diff
@@ -14,8 +15,62 @@ by leveraging the modern web.
 ## API Spec
 The preferred JSON object to be returned by the API should be structured as follows:
 
-### Users (for authentication)
+## Links to the Mock up Design
+1. ### Sign Up, Log In and Create Article
+    [Full deck link](https://wireframepro.mockflow.com/view/M15c07edf891b45f0dfaa1f5cc85c68c61547577768098#/page/D31452032c99b79d208bf34f194933048)
+
+2. ### Homepage, Landing page and Profile
+   [Full deck link](https://wireframepro.mockflow.com/view/M724c1af1281229c8914e2f1e3b9e2a201547621799851)
+
+
+## Hosting
+Staging-https://ah-fulldeck-staging.herokuapp.com/
+
+
+## How should this be manually tested?
+
+1. Install pip first using this command `sudo apt-get install python3-pip`
+1. Then install virtualenv using pip3 `sudo pip3 install virtualenv `
+1. Now create a virtual environment `virtualenv venv `
+1. Activate your virtual environment: `source venv/bin/activate`
+1. Clone this repo with the following command:
+`git clone https://github.com/andela/ah-fulldeck.git`
+1. Cd into the ah-fulldeck directory:
+`cd ah-fulldeck`
+1. Install the requirements by running the command:
+`pip install requirements.txt`
+1. Create a database:
+  - In the terminal run the following after installing postgres and setting up:  
+  `psql`
+  
+    `create DATABASE <name_of_your_database>`
+
+1. Export the environment settings as follows:  
+`export DATABASE="<name of your database>"`  
+   
+   `export USER="<your postgres username>"`  
```

Comment:
3. Number 1 to 8 not well spaced, And how does number 1 come after number 8? Number 2 (The second number 2) seems well organised, use the same structure for all other instructions

Reply:
@Hesbon5600 has clarified that. Its markdown styling. If you check the Readme its well numbered.